cms/djangoapps/contentstore/tests/utils.py

```diff
@@ -100,6 +100,18 @@ def create_non_staff_authed_user_client(self, authenticate=True):
         nonstaff.is_authenticated = lambda: authenticate
         return client, nonstaff
 
+    def create_staff_authed_user_client(self, authenticate=True):
```

Comment:
`self.client` should already be a staff-authed client (see `class ModuleStoreTestCase` definition), so this method is not needed.